.github/workflows/dotnet.yaml

```
@@ -125,9 +125,12 @@ jobs:
 
   all-required-checks-complete:
     needs: [check-format, build, expected-pack, analyze-code, github-tag-and-release-dry-run]
+    if: ${{ always() }}
```

Not needed for `if:`:
```suggestion
    if: always()
```

---

TIL!